bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -208,16 +290,12 @@ class VideoPreview extends Component {
 
     // logger.debug('starting scan');
 
-    const checkWebcamExists = () => {
-      // logger.debug('initial webcam check');
-      // we call gUM with no constraints so we know if any stream is available
-      this.doGUM(deviceId, {}).then((stream) => {
-        if (!this._isMounted) return;
+    const scanningCleanup = () => {
```

Comment:
This function is defined again in: https://github.com/bigbluebutton/bigbluebutton/blob/9c8a616ef3ad0b7d6f8a25f062b4406f8cb41fcb/bigbluebutton-html5/imports/ui/components/video-preview/component.jsx#L406

Causing a problem:
```
=> Errors prevented startup:

   While building for web.browser:
   imports/ui/components/video-preview/component.jsx: /home/bbb/dev/bigbluebutton/bigbluebutton-html5/imports/ui/components/video-preview/component.jsx: Duplicate declaration "scanningCleanup"
     404 |     };
     405 |
   > 406 |     const scanningCleanup = () => {
         |           ^
     407 |       this.video.onloadedmetadata = undefined;
     408 |
     409 |       if (availableProfiles.length > 0) {

=> Your application has errors. Waiting for file change.
=> Modified -- restarting.
=> Errors prevented startup:

   While building for web.browser:
   imports/ui/components/video-preview/component.jsx: /home/bbb/dev/bigbluebutton/bigbluebutton-html5/imports/ui/components/video-preview/component.jsx: Duplicate declaration "scanningCleanup"
     404 |     };
     405 |
   > 406 |     const scanningCleanup = () => {
         |           ^
     407 |       this.video.onloadedmetadata = undefined;
     408 |
     409 |       if (availableProfiles.length > 0) {

=> Your application has errors. Waiting for file change.
```